### PR TITLE
Pw space [depends on mono font]

### PR DESCRIPTION
**Pull request by NickeZ**

### CODE REVIEW

**Comment by benma** [comment]
You are my hero :heart_eyes: 

In unlock.c in the confirmation of the passphrase, you can also use this font and remove the checks and warnings involving spaces, since they would become visible.

**Comment by NickeZ** [comment]
> You are my hero heart_eyes

Life goal achieved ;)

> 
> In unlock.c in the confirmation of the passphrase, you can also use this font and remove the checks and warnings involving spaces, since they would become visible.

Done

File: src/ui/fonts/password_9X9.c [hunk 1]
```diff
@@ -0,0 +1,108 @@
+// Converted from password.ttf
```
Comment (benma) [inline src/ui/fonts/password_9X9.c]: Would be useful to commit this file as well? Eventually those files could be generated at build time from the ttfs.

Comment (NickeZ) [reply src/ui/fonts/password_9X9.c]: That is a "search and replace" effect. The original file is the one referenced in the normal "a" fonts and that file is lost afaik. This comment should be updated to mention that it is almost a copy of the "font_a" c/h files.

Comment (NickeZ) [reply src/ui/fonts/password_9X9.c]: Updated the comment.

**Pull request merged**